src/ConfigUtil.ts

```diff
@@ -359,7 +359,7 @@ export class ConfigUtil {
                 (target[key] as {}) = {};
             }
 
-            //@ts-ignore Expression produces a union type that is too complex to represent.
+            // @ts-ignore Expression produces a union type that is too complex to represent.
```

Comment:
Can we fix this ts-ignore?

Reply:
I dont think we can this time.  A few lines above there is this comment 

```
        // The types get a bit complicated here, so a few "any"'s are required to prevent compile errors...
        // TypeScript can't figure out the type of T[key], since 'key' is a runtime argument.
```